fpdcc/document-search
# Allow users to reset their passwords

## Overview

This branch sets up urls and templates for resetting a user's password. There is a new "Forgot your password" link on the login page that will send users an email with a link to reset their password.

Closes #43

### Demo

![image](https://github.com/fpdcc/document-search/assets/114717958/91f69878-9250-4d1f-9ee1-6ef9e0426e21)

## Testing Instructions

* Navigate to the login page and click "Forgot your password?"
* Enter the email associated with your account
* Check your email for instructions and a link to reset
* On the reset form, enter invalid and valid inputs according to the instructions shown
  * Confirm that invalid inputs return and display errors
  * Confirm that valid inputs redirect you to a success page
* Log in with your new password
